.travis.yml

```diff
@@ -12,7 +13,7 @@ script:
 - yarn build-ppx
 - yarn build-lib
 - npm test
-- mv ppx_src/_esy/default/build/default/.ppx/ppx_decco/ppx.exe ppx-$TRAVIS_OS_NAME.exe
+- mv ppx_src/_esy/default/build/install/default/lib/ppx_decco/ppx.exe ppx-$TRAVIS_OS_NAME.exe
```

Comment:
I'm not sure this is correct, when I tried building it that `ppx.exe` is a symlink.